text/0000-href-to.md

```diff
@@ -0,0 +1,47 @@
+- Start Date: 2015-09-12
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes the addition of a `href-to` helper which is a lightweight alternative to the `link-to` helper.
+
+# Motivation
+
+For performance sensitive applications like [Intercom](https://www.intercom.io/), `link-to` can be quite heavy. We've found many cases where all we want is to link to a route. `link-to` creates a component with a set of attribute and class bindings which we don't need. In contrast, the `href-to` helper would simply render a URL at render time.
```

Comment:
Yeah, the faster that `link-to` is, the less of a need for something like this. We've found the `href-to` addon to be 6-12x times faster in Ember 1.11-1.13 though. `link-to` will always have much more to do, all `href-to` has to do is to generate a URL.